package.json

```diff
@@ -305,7 +307,7 @@
         },
         "peripheral-inspector.svdAddrGapThreshold": {
           "type": "number",
-          "default": 16,
+          "default": -1,
```

Comment:
Setting to `-1` is useful for testing purposes as discussed "offline". But we should not change the default on `main`.